src/app/pages/system/advanced/advanced.component.ts

```diff
@@ -65,7 +65,11 @@ export class AdvancedComponent implements OnDestroy {
                   if (this.dialogRef) {
                     this.dialogRef.close();
                   }
-                  this.dialog.errorReport(helptext_system_advanced.debug_download_failed_title, helptext_system_advanced.debug_download_failed_message, err);
+                  if(err instanceof HttpErrorResponse) {
+                    this.dialog.errorReport(helptext_system_advanced.debug_download_failed_title, helptext_system_advanced.debug_download_failed_message, err.message);
+                  } else {
+                    this.dialog.errorReport(helptext_system_advanced.debug_download_failed_title, helptext_system_advanced.debug_download_failed_message, err);
```

Comment:
Will it still be `[object Object]` for "connection reset by peer", network timeout, etc?

Reply:
We've tried to cover all the possible scenarios here. Both http errors and websocket errors are being handled separately. We should be good.